src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationReferenceBuilder.java

```diff
@@ -218,4 +218,22 @@ public static String referenceForClaimant(String claimReferenceNumber) {
             return referenceForClaim(claimReferenceNumber, CLAIMANT);
         }
     }
+
+    public static class MediationSuccessful {
+
+        public static final String TEMPLATE = "to-%s-mediation-successful";
```

Comment:
Does this need to be exposed? Change to `private` if not

Reply:
I am not sure, Ellie said this has something to do with gov notify